eng/WpfArcadeSdk/tools/Wpf.Cpp.targets

```diff
@@ -410,11 +419,24 @@ using namespace System::Runtime::Versioning;
       </ProjectToBuild>
     </ItemGroup>
     <MSBuild Projects="@(ProjectToBuild)"
-             Targets="Restore;ResolveReferences;IdentifyNugetReferences"
+             Targets="Restore"
+             Properties="TargetFramework=$(TargetFramework);Platform=$(Architecture);RuntimeIdentifier=win-$(Architecture);ForceEvalOnNextMsBuild=true" />
```

Comment:
This looks hacky why was this required?

Reply:
This is part of the changes to get the nuget package path variables hooked into the build where I needed them. A lot of this was to support getting the vars into vcxprojs so I could properly reference them (EX: the netstandard System.IO.Packaging)